File: C/Prefix Sum/1732. Find the Highest Altitude.c
/* 1732. Find the Highest Altitude
Difficulty: Easy
Submission Link: https://leetcode.com/problems/find-the-highest-altitude/submissions/1379774149/ */

int largestAltitude(int* gain, int gainSize){
    int max = 0;
    int temp = 0;
    for(int i=0;i<gainSize;i++){
        temp += gain[i];
        if(temp > max){
            max = temp;
        }
    }
    return max;
}

File: C#/Prefix Sum/1732. Find the Highest Altitude.cs
/* 1732. Find the Highest Altitude
Difficulty: Easy
Submission Link: https://leetcode.com/problems/find-the-highest-altitude/submissions/1379774043/ */

public class Solution {
    public int LargestAltitude(int[] gain) 
    {
             
        int deger=0;
        int max = 0;
        for(int i = 0; i < gain.Length; i++)
        {
            deger = deger + gain[i];
            if(max<deger) max = deger;          
        }
    
    return max;
    }
}